lighthouserc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "ci": {
+      "collect": {
+        "url": ["http://localhost:4173"],
+        "startServerCommand": "bun run start | sed -u 's/^/[server] /' & pid=$!; sleep 2; echo 'Server is ready'; wait $pid",
```

Comment:
Lighthouse CI uses "Server is ready" echo command to indicate when to start analysis.